src/commcare_cloud/ansible/deploy_db.yml

```diff
@@ -1,194 +1,18 @@
 ---
-- name: Common Database Machine Setup
-  hosts:
-    - postgresql
-    - couchdb
-    - couchdb2
-    - redis
-    - elasticsearch
-    - rabbitmq
-    - zookeeper
-    - riakcs
-    - formplayer
-    - pg_standby
-  become: true
-  roles:
-    - {role: ecryptfs, tags: 'ecryptfs'}
-    - {role: backups, tags: 'backups'}
-  tasks:
-    - import_tasks: roles/ecryptfs/tasks/remove-tmp.yml
-      tags: ecryptfs-cleanup
-
-- name: PostgreSQL
-  hosts: postgresql:pg_standby
-  become: true
-  roles:
-    - {role: postgresql, tags: 'postgresql'}
-
-- name: PostgreSQL Backup
-  hosts: postgresql:pg_backup
-  become: true
-  roles:
-    - {role: pg_backup, tags: ['postgresql', 'backups', 'pg_backup']}
-
-- name: Setup auth for standby
-  hosts:
-    - postgresql
-  vars:
-    - userFrom: postgres
-    - hostTo: "{{ hot_standby_server }}"
-    - userTo: postgres
-    - state: 'present'
-  tasks:
-    - import_tasks: roles/setup_auth_keys.yml
-      when: hot_standby_server is defined
-  tags:
-    - pg_standby
-
-- name: Couchdb
-  hosts: couchdb
-  become: true
-  roles:
-    - {role: couchdb, tags: 'couchdb'}
-
-- name: Couchdb log rolling configurations
-  hosts: couchdb
-  become: true
-  roles:
-    - role: ansible-logrotate
-      tags: couchdb
-      logrotate_scripts:
-        - name: "{{ deploy_env }}_couchdb"
-          path: "{{ couch_log_dir }}/*.log"
-          options:
-            - daily
-            - size 100M
-            - rotate 10
-            - missingok
-            - compress
-            - delaycompress
-            - copytruncate
-            - nocreate
-            - notifempty
-
-- name: Couchdb 2.0
-  hosts: couchdb2
-  become: true
-  roles:
-    - {role: couchdb2, tags: 'couchdb2'}
-
-- name: Couchdb2 log rolling configurations
-  hosts: couchdb2
-  become: true
-  roles:
-    - role: ansible-logrotate
-      tags: couchdb2
-      logrotate_scripts:
-        - name: "{{ deploy_env }}_couchdb2"
-          path: "{{ couchdb_dir }}/var/log/*.stderr"
-          options:
-            - daily
-            - size 100M
-            - rotate 10
-            - missingok
-            - compress
-            - delaycompress
-            - copytruncate
-            - nocreate
-            - notifempty
-
-- name: Redis
-  hosts: redis
-  become: true
-  roles:
-    - {role: redis, tags: 'redis'}
-
-- name: Redis Monitoring
-  hosts: redis
-  become: true
-  roles:
-    - redis_monitoring
-
-- name: Elasticsearch
-  hosts: elasticsearch
-  become: true
-  roles:
-    - {role: elasticsearch, tags: 'elasticsearch'}
-
-- name: RabbitMQ
-  hosts: rabbitmq
-  become: true
-  roles:
-    - {role: rabbitmq, tags: 'rabbitmq'}
-
-- name: Zookeeper
-  hosts: zookeeper
-  become: true
-  roles:
-    - {role: zookeeper, tags: 'zookeeper'}
-
-- name: Kafka
-  hosts: kafka
-  become: true
-  environment:
-    JMX_PORT: 9999
-  roles:
-    - {role: kafka, tags: 'kafka'}
 
+- import_playbook: update_apt_cache.yml
```

Comment:
i don't see a new file added for this one in this pr, is this adding something new to this list of steps that wasn't here before?

also i see you created a new file for `common_db_machine_setup.yml` in this pr but i don't see a reference it from here. is that intentional?

Reply:
Oops, copy and paste error. Put in the correct playbook name: https://github.com/dimagi/commcare-cloud/pull/2002/commits/5a9be422713688b010090e338628e8626247ebcf